.spelling

```diff
@@ -13,8 +13,10 @@ DevOps
 dojo
 e.g.
 i.e.
+Kaizen
 Katacoda
 nvie
+VSM
```

Comment:
All capitals should not be necessary, as it is an acronym and should be ignored by default.

Reply:
changed